packages/docs-browser/src/components/DocsBrowser/DocsBrowser.js

```diff
@@ -101,4 +101,8 @@ DocsBrowser.propTypes = {
   hasLeftPadding: PropTypes.bool
 }
 
+DocsBrowser.defaultProps = {
```

Comment:
you should set the initial value in the store. (see reducer.js file with hasLeftèadding in it)